### PR TITLE
Fix/heat params

### PR DESCRIPTION
**Pull request by MaGering**
With this PR the pvcompare input parameters are adjusted to assumptions that correspond more to realistic operation conditions
Explanations have been added to the rtd.

The following steps were realized, as well (required):
- [x] Update the CHANGELOG.md
- [x] Check if full simulation tests pass locally (`EXECUTE_TESTS_ON=master pytest`)

Also the following steps were realized (if applies):
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [ ] For new functionalities: Explain in readthedocs
- [ ] Write test(s) for your new patch of code

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

In case of an error due to linting, run `black . --exclude docs/` and push your changes.
Note that in case you do not fix a whole issue you should start your PR with `Address #xyz`.

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by MaGering** [comment]
I don't understand why the check does not pass for `tests/test_plots.py`. Locally it does. Could you help me with this @Piranias, @SabineHaas?

**Comment by Piranias** [comment]
> I don't understand why the check does not pass for `tests/test_plots.py`. Locally it does. Could you help me with this @Piranias, @SabineHaas?

I also don't know where this is coming from, I had the same issue in PR #273 and wanted to ask you two about it..

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ Here is a template for new release sections
 ### Added
 -
 ### Changed
--
+- The pvcompare input parameters have been revised (#272)
```
Comment (SabineHaas) [inline CHANGELOG.md:27]: Could you be a bit more specific here?

**Review by SabineHaas — APPROVED** [review]

**Comment by SabineHaas** [comment]
Ah, I've already approved but you should not merge before #273 is merged.

File: CHANGELOG.md (hunk 1 above)
Comment (MaGering) [reply CHANGELOG.md]: Yes of course, I specified what I did in commit https://github.com/greco-project/pvcompare/pull/272/commits/e3e286e46f67f963f0e5bf71bfc5e95ae34568a3.

**Comment by MaGering** [comment]
@SabineHaas I adapted the parameters of the stratified thermal storage, because they based on insufficient assumptions.

**Pull request merged**